service/instance.py

```diff
@@ -1345,17 +1346,98 @@ def security_group_init(core_identity, max_attempts=3):
     return user_security_group_init(core_identity)
 
 
-def user_security_group_init(core_identity, security_group_name='default'):
-    # Rules can come from the provider _or_ from settings _otherwise_ empty-list
-    rules = core_identity.provider.get_config('network', 'default_security_rules',getattr(settings,'DEFAULT_RULES',[]))
-    driver = get_cached_driver(identity=core_identity)
-    lc_driver = driver._connection
-    security_group = get_or_create_security_group(lc_driver, security_group_name)
-    set_security_group_rules(lc_driver, security_group, rules)
+def user_security_group_init(core_identity, security_group_name='atmosphere'):
```

Comment:
Just one thing I want to point out is that this function was using `default` security group. I change it to use a new security group called atmosphere and that's why I add `return {"ex_metadata": ex_metadata, "ex_keyname": ex_keyname, "ex_security_groups": ex_security_groups}`